Procfile

```
@@ -0,0 +1,2 @@
+web: bin/rails server -p $PORT -e $RAILS_ENV
```

This should actually use puma

```suggestion
web: bundle exec puma -C config/puma.rb
```

https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server#procfile